data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
sorry, i'm not too familiar yet with your rearchitecting of all the infos. Can you explain what data-common is supposed to represent?